clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/handlers/TitusDeployHandler.groovy

```diff
@@ -176,6 +182,9 @@ class TitusDeployHandler implements DeployHandler<TitusDeployDescription> {
           //migrationPolicy should only be used when the disruptionBudget has not been specified
           description.migrationPolicy = description.migrationPolicy ?: sourceJob.migrationPolicy
         }
+        if (description.disruptionBudget == null && description.migrationPolicy == null) {
+          description.disruptionBudget = getDefaultDisruptionBudget()
```

Comment:
I added this check earlier , do we need to get rid of it now ? #3553

Reply:
No, the changes from #3553 are still valid! We only want to set the default if there is no migration policy or disruption budget defined.

Reply:
ok , LGTM